Add 'didFailToLoadPlacementNamed' example.

pcimring-taboola opened the pull request:
1. Add 'didFailToLoadPlacementNamed' example to sample TBLWebDelegate code.
2. Add some comments to the code.
3. Clean up the 'onItemClick' example.
4. Add .gitignore file. (Added unintentionally, but perhaps worth keeping? :)

pcimring committed:
Add 'didFailToLoadPlacementNamed' example.

1. Add 'didFailToLoadPlacementNamed' example to sample TBLWebDelegate code.
2. Add some comments to the code.
3. Clean up the 'onItemClick' example.

omriariav commented:
@pcimring-taboola - contact @tzufitltaboola to approve this PR
Well done

tzufitltaboola commented on .gitignore:
```diff
@@ -0,0 +1,217 @@
+#########################
```
I think you should remove this file you added.

pcimring committed:
Remove .gitignore

1) Remove .gitignore file
2) Update Podfile for Objective-C project

pcimring committed:
Fix sample code for 'didLoadPlacementNamed' event (Web Integration)

1) In the WebViewController class, replace the local variable with a TBLWebPage member variable (for Swift and Objective C).
2) Make minor changes to the code comments.
